user.css

```diff
@@ -1593,26 +1593,23 @@ option {
 
 /* play buttons */
 
-.main-playButton-PlayButton,
-.main-playPauseButton-button,
-.main-trackList-rowPlayPauseButton {
-  padding: 10px;
```

Comment:
Looks like this padding was the main issue :)

The other changes just style the play/pause buttons.